apps.yml

```diff
@@ -4033,3 +4033,14 @@ apps:
     name: health.telemetry.mozilla.org
     op: auth0
     url: https://health.telemetry.mozilla.org/openid/callback/login
+- application:
+    authorized_groups:
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: Nqo9YvvVxjpLDLPv7YGmpJXv2JNraUtH
+    display: false
+    logo: smartling-logo.png
```

Comment:
You have `smartling-logo.png` here, but your file is named `smartling-log0.png`.

(also, it could probably just be `smartling.png`)

Otherwise, everything looks fine.